### ISSUE TITLE
Long text beside Icon doesnt show full text

### ISSUE THREAD

**Issue by ahmedkassem** [GeekyAnts/NativeBase#124]
In the list component when using an icon, with a long text ListItem , only part of the text is shown, not the full length text
same goes for card component in title with icon

**Comment by sankhadeeproy007** [comment]
Please attach a screenshot and code sample.

**Comment by ahmedkassem** [comment]
Code
```
<List>
                        <ListItem  iconLeft onPress={Actions.singlenews}>
                            <Icon Large style={{color:'#e5029a'}} name="ios-notifications" />
                            <Text style={{color:'#e5029a'}}>Short Title</Text>
                        </ListItem>
                        <ListItem iconLeft onPress={Actions.singlenews}>
                            <Icon Large style={{color:'#039BE5'}} name="ios-notifications" />
                            <Text style={{color:'#039BE5'}}>Long Title Long Title Long Title Long Title Long Title </Text>
                        </ListItem>
                        <ListItem iconLeft onPress={Actions.singlenews}>
                            <Icon name="ios-notifications" />
                            <Text>Read News</Text>
                        </ListItem>
</List>
```

![screen1](https://cloud.githubusercontent.com/assets/7607004/17837097/88186e8e-67a9-11e6-96e8-635df45ea1d9.jpg)

**Comment by sankhadeeproy007** [comment]
Fixed with 0.5.6
